builtins/amp-video.js

```diff
@@ -49,24 +69,23 @@ export function installVideo(win) {
         assertHttpsUrl(this.element.getAttribute('src'), this.element);
       }
       this.propagateAttributes(
-          ['src', 'controls', 'autoplay', 'muted', 'loop', 'poster'],
-          video);
-      video.width = getLengthNumeral(width);
-      video.height = getLengthNumeral(height);
-      this.applyFillContent(video, true);
+          ['src', 'controls', 'autoplay', 'muted', 'loop', 'preload'],
+          this.video_);
+
       this.getRealChildNodes().forEach(child => {
+        // Skip the video we already added to the element.
+        if (this.video_ === child) {
+          return;
+        }
         if (child.getAttribute && child.getAttribute('src')) {
           assertHttpsUrl(child.getAttribute('src'), child);
         }
-        video.appendChild(child);
+        this.video_.appendChild(child);
       });
-      this.element.appendChild(video);
 
-      /** @private {?HTMLVideoElement} */
-      this.video_ = video;
-      setStyles(video, {visibility: 'hidden'});
-      return loadPromise(video).then(() => {
-        setStyles(video, {visibility: ''});
+      setStyles(this.video_, {visibility: 'hidden'});
```

Comment:
Why are we setting `hidden` here? (I know this was here before). This will also hide the poster which would somewhat negative any benefit from this PR.